src/stories/plugins/CodeBlockUI.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import ButtonView from '@ckeditor/ckeditor5-ui/src/button/buttonview';
+import Plugin from '@ckeditor/ckeditor5-core/src/plugin';
+import codeBlockIcon from '@zendeskgarden/svg-icons/src/16/terminal-window-stroke.svg';
+import { getNormalizedAndLocalizedLanguageDefinitions } from '@ckeditor/ckeditor5-code-block/src/utils';
+
+export class CodeBlockUI extends Plugin {
+  init() {
+    const editor = this.editor;
+    const t = editor.t;
+    const componentFactory = editor.ui.componentFactory;
+    const normalizedLanguageDefs = getNormalizedAndLocalizedLanguageDefinitions(editor);
+    const defaultLanguageDefinition = normalizedLanguageDefs[0];
+
+    componentFactory.add('codeBlock', locale => {
+      const command = editor.commands.get('codeBlock');
+
+      const buttonView = new ButtonView(locale);
+
+      buttonView.set({
+        label: t('Insert code block'),
```

Comment:
I'm not too familiar with localization but I presume it isn't necessary for this label? Or is it?

Reply:
Using the translate utilities allows this to be translated by consumers. [Localization API](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/deep-dive/ui/localization.html#writing-a-localizable-ui)

But, with this being a static example we aren't swapping locales here.

Reply:
Oh right, I forgot this was an example 👍🏽